src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java

```diff
@@ -497,6 +499,8 @@ private void deparseTableName(String tableName, StringBuilder tableNameWithAlias
             tableNameWithAlias.append(SpellingMistake.useOrDefault(this.tableNameSpellingMistake, tableName ));
             tableNameWithAlias.append("|");
             tableNameWithAlias.append(SpellingMistake.useOrDefault(this.tableNameSpellingMistake, this.getRelatedTableNameOrAlias(tableName)));
+            tableNameWithAlias.append("|");
+            tableNameWithAlias.append("\\w*?");
```

Comment:
Why it is allowed to insert any word as a table name in Expressions?

This would allow something like, wouldn't it?:
SELECT col1 FROM table1 WHERE Butterbrot.col1 = 5

Reply:
done